src/main/java/com/epam/ld/module2/testing/email/EmailConfiguration.java

```diff
@@ -0,0 +1,72 @@
+package com.epam.ld.module2.testing.email;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.support.ResourceBundleMessageSource;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+
+import java.util.Properties;
+
+@Configuration
+@ConfigurationProperties(prefix = "com.epam.l3.messenger")
+public class EmailConfiguration {
+
+    @Value("${spring.mail.host}")
+    private String mailServerHost;
+
+    @Value("${spring.mail.port}")
+    private Integer mailServerPort;
+
+    @Value("${spring.mail.username}")
+    private String mailServerUsername;
+
+    @Value("${spring.mail.password}")
+    private String mailServerPassword;
+
+    @Value("${spring.mail.properties.mail.smtp.auth}")
+    private String mailServerAuth;
```

Comment:
during previous review we discussed to separate fields from methods on configuration, isn't it?